### ISSUE TITLE
[hue] Missing or invalid configuration  / Hue ambient light sensor / openHAB 3.3M1

### ISSUE THREAD

**Issue by michaeljoos72** [openhab/openhab-addons#12186]
## Expected Behavior
Thing should stay online without error. Never had an issue with this in openHAB 3.2

## Current Behavior
**1. openHAB auto discover does work for light sensor (motion sensor) but shows error in log:**
![image](https://user-images.githubusercontent.com/20517474/152042594-301dec80-bfa6-47ed-9f09-039d4d806518.png)

Debug-Log:
2022-02-01 21:13:44.341 [DEBUG] [.discovery.HueDeviceDiscoveryService] - discovered unsupported sensor of type 'Daylight' and model 'PHDL00' with id 1
2022-02-01 21:13:44.896 [DEBUG] [.discovery.HueDeviceDiscoveryService] - discovered unsupported sensor of type 'Daylight' and model 'PHDL00' with id 1

**2. Add thing(s) does show status ONLINE:**
![image](https://user-images.githubusercontent.com/20517474/152042772-6e0bb824-27e5-4f01-822c-0b2f1d9f4052.png)

**3. After change/save/reboot Thing(s) are in status UNINITIALIZED / HANDLER_CONFIGURATION_PENDING**
![image](https://user-images.githubusercontent.com/20517474/152043176-934765cf-a567-4463-8858-77bbcd3d6fed.png)
![image](https://user-images.githubusercontent.com/20517474/152043251-d9a5481a-a63b-4d53-a908-3269f6d67a51.png)

Debug-Log after reboot:
[openhab.log](https://github.com/openhab/openhab-addons/files/7981684/openhab.log)

## Your Environment
openHABian v1.7.1 / Raspberry Pi OS Bullseye
openHAB 3.3.0.M1 - Milestone Build

java -version
openjdk version "11.0.14" 2022-01-18
OpenJDK Runtime Environment (build 11.0.14+9-post-Raspbian-1deb11u1)
OpenJDK Server VM (build 11.0.14+9-post-Raspbian-1deb11u1, mixed mode)

**Comment by jlaur** [comment]
@cweitkamp - seems similar to #12182?

**Comment by cweitkamp** [comment]
Yes, this issue has been addressed with #12182.

As a workaround you can go to the thing -> code section and set the value for the `ledindication` parameter to `true` or `false`.

```
thingTypeUID: hue:0106
configuration:
  tholddark: 12992
  ledindication: false
  tholdoffset: 7000
  on: true
```
